### ISSUE TITLE
--cached-build from a defined location

### ISSUE THREAD

**Issue by Batistleman** [zodern/meteor-up#1065]
**tldr;**

> Would it be possible to specify the --cached-build location through the command line?

I see that the buildOptions.buildLocation ([link](https://github.com/zodern/meteor-up/blob/3c7120a75c12ea12fdd5688e33574c12e158fd07/src/plugins/meteor/command-handlers.js#L103)) can be specified, but I would like deploy the built bundle to multiple configurations.

I think it would also be useful to store all the built bundles to be able to do a quick rollback if needed.

__My current way of working:__

1. I checkout master 
1. I grab the commit hash and put it in a file in the app folder (so it's available in the client interface, and I can see what commit version is deployed)
1. cd to the following directories (depending on what we want deploy): staging, production1, customer 1, customer 2, etc. (each of these folders has its own mup.js and settings.json files)
1. I run `npx mup deploy` in each one

__But what I would really like is a 'mup-deployer' tool that takes the following parameters:__
* --use-commit <hash>
   * if --use-commit is not specified, use head
   * if <hash> was previously built, we can use the cached bundle (as nothing can change I guess?)
* --mup-configs staging,production1,customer1 
   * the mup configurations you want to deploy. eg. staging.mup.js and staging.settings.json

It would store the preformed deploys in a log file, so we can see what machines have which version 
It would also need a config file to store the git details and application.
(Nice to have: an internal interface to see the status of each machine)

I guess most this can be scripted, but I'm wondering if anyone would find this useful as well...?
